dev-util/pycharm-professional/pycharm-professional-2017.3.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils readme.gentoo-r1
```

Comment:
I think you need no eutils here https://blogs.gentoo.org/mgorny/2015/11/13/the-ultimate-guide-to-eapi-6/